catppuccin.tmux

```diff
@@ -31,6 +31,160 @@ setw() {
   tmux_commands+=(set-window-option -gq "$option" "$value" ";")
 }
 
+build_window_icon() {
+  local window_status_icon_enable="$(get_tmux_option "@catppuccin_window_status_icon_enable" "yes")"
+
+  local custom_icon_window_last="$(get_tmux_option "@catppuccin_icon_window_last" "󰖰")"
+  local custom_icon_window_current="$(get_tmux_option "@catppuccin_icon_window_current" "󰖯")"
+  local custom_icon_window_zoom="$(get_tmux_option "@catppuccin_icon_window_zoom" "󰁌")"
+  local custom_icon_window_mark="$(get_tmux_option "@catppuccin_icon_window_mark" "󰃀")"
+  local custom_icon_window_silent="$(get_tmux_option "@catppuccin_icon_window_silent" "󰂛")"
+  local custom_icon_window_activity="$(get_tmux_option "@catppuccin_icon_window_activity" "󰖲")"
+  local custom_icon_window_bell="$(get_tmux_option "@catppuccin_icon_window_bell" "󰂞")"
+
+  if [[ $window_status_icon_enable == "yes" ]]
+  then
+    local show_window_status="#(printf '%%s\n' '#F' | sed \"s/*/${custom_icon_window_current}/\" | sed \"s/-/${custom_icon_window_last}/\" | sed \"s/#/${custom_icon_window_activity}/\" | sed \"s/#//g\"| sed \"s/~/${custom_icon_window_silent}/\" | sed \"s/!/${custom_icon_window_bell}/\" | sed \"s/M/${custom_icon_window_mark}/\" | sed \"s/Z/${custom_icon_window_zoom}/\")"
+  fi
+
+  if [[ $window_status_icon_enable == "no" ]]
```

Comment:
`[[` and `==` will only work in Bash, not in POSIX shells. Several Linux distros such as Alpine and all BSDs do not come with Bash. Additionally, if the variable is empty for some reason, an error will occur. It should be quoted to avoid this. This can be written more portably as `[ "$foo" = "bar" ]`.